src/main/java/org/apache/sling/distribution/agent/impl/SimpleDistributionAgentQueueProcessor.java

```diff
@@ -92,7 +92,7 @@ public boolean process(@NotNull String queueName, @NotNull DistributionQueueEntr
 
             final long endTime = System.currentTimeMillis();
 
-            distributionLog.debug("[{}] ITEM-PROCESSED item={}, status={}, processingTime={}ms", queueName, queueItem, success, endTime - startTime);
+            distributionLog.info("[{}] ITEM-PROCESSED item={}, status={}, processingTime={}ms", queueName, queueItem, success, endTime - startTime);
```

Comment:
We should not change the log level as it would lead to a lot of additional log messages.

Reply:
@cschneider thanks for the review. My motivation behind the change was to also log the queueItemID in case of successful processing of item. The item ID is currently only logged in debug level or at error/warn level in case of an exception. I have updated the existing log info for successful distribution of item and added the queue item ID to it - https://github.com/apache/sling-org-apache-sling-distribution-core/pull/55/commits/33b54a5a56381fd7a3c60db46d6001cbf5426780

Would this be an acceptable change?

Reply:
This is what the distribution lon entry would look like now in case of successful distribution - 

`2021/09/14 13:21:02:084 - INFO - [queue-bpdistributionagent0] PACKAGE-DELIVERED DSTRQ2: ADD item=dstrpck-1631605857371-3241eb55-0949-4ece-953d-4ebbf2ebf81d, paths=[/content/dam/10683CD3.JPG], importTime=4627ms, execTime=4739ms, size=10611B`